v1/account/all.md

```diff
@@ -577,26 +687,26 @@ curl -X PUT \
 	}'
 ```
 
-### HTTP Request
+### HTTP Request {#http-request-method-put-updateaccount}
 
 `PUT https:///v1/accounts/{AccountID}`
 
-### Query Parameters
+### Query Parameters {#query-parameters-method-put-updateaccount}
 
 | Name       | Type   | Description                                                  |
 |------------|--------|--------------------------------------------------------------|
 | account_id | string | AccountID is the unique identifier of the account to update. |
 
-### Body Parameters
+### Body Parameters {#body-parameters-method-put-updateaccount}
 
 | Name        | Type   | Description                                                  |
 |-------------|--------|--------------------------------------------------------------|
 | account_id  | string | AccountID is the unique identifier of the account to update. |
 | description | string | Description to update                                        |
```

Comment:
Some fields have period at the end some fields aren't, let it be consistent.

Reply:
It doesn't seem to be related to this PR, should be fixed in a separate PR later.